src/main/java/io/druid/data/input/impl/SpatialDimensionSchema.java

```diff
@@ -18,12 +18,14 @@
 package io.druid.data.input.impl;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.List;
 
 /**
  */
+@Deprecated
 public class SpatialDimensionSchema
 {
   private final String dimName;
```

Comment:
i see, but then, what should users start doing differently going forward so that we can change/remove SpatialDimensionSchema eventually given that its deprecated?